src/components/20-molecules/image-upload/index.react.d.ts

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+interface AXAImageUploadProps {
+  inputFileText?: String;
```

Comment:
We written the types in all components lowercase

Reply:
done